test-files/cmd/slow/DeriveChangesetCmdTest.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
DeriveChangesetCmdTest has more in it now, so needed to be moved from quick to slow.